kinto/plugins/kinto_admin/tests/__init__.py

```diff
@@ -0,0 +1,26 @@
+import os
+
+import webtest
+from kinto.core import utils as core_utils
+
+
+def get_user_headers(user):
+    credentials = "%s:secret" % user
+    authorization = 'Basic {0}'.format(core_utils.encode64(credentials))
+    return {
+        'Authorization': authorization
+    }
+
+
+class BaseWebTest(object):
+    config = '../../../../config/kinto.ini'
```

Comment:
We should use the test config file instead. You can even create a specific one for the plugin tests. What we want is https://github.com/mozilla-services/kinto-dist/blob/master/config/kinto.ini#L133

You can also configure tests like that https://github.com/Kinto/kinto/blob/master/tests/plugins/test_history.py#L29-L32 rather than using a config file.